packages/react/src/components/ProgressBar/ProgressBar-story.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState, useEffect } from 'react';
+
+import { withKnobs, text, boolean, number } from '@storybook/addon-knobs';
+import ProgressBar from '../ProgressBar';
+
+const props = () => ({
+  label: text('Label text (label)', 'Progress bar label'),
+  helperText: text('Helper text (helperText)', 'Optional helper text'),
+  hideLabel: boolean('Hide the label (hideLabel)', false),
+  value: number('Current value (value)', 75),
+  max: number('Maximum value (max)', 100),
+});
+
+export default {
+  title: 'Experimental/unstable_ProgressBar',
+  decorators: [withKnobs],
+
+  parameters: {
+    component: ProgressBar,
+  },
+};
+
+export const _ProgressBar = () => (
+  <ProgressBar
+    label="Progress bar label"
+    helperText="Optional helper text"
+    value={75}
+  />
+);
+_ProgressBar.storyName = 'ProgressBar';
```

Comment:
It's just because `ProgressBar` is already defined as the component import, so otherwise I'd need to `import ProgressBarComponentfrom '../ProgressBar'` to avoid the conflict which may cause confusion when devs look at thr story source.

Reply:
@janhassel ah! yes of course, sorry I missed that.